multisrc/overrides/madara/randomscan/src/luratoon.kt

```diff
@@ -7,9 +7,9 @@ import java.text.SimpleDateFormat
 import java.util.Locale
 import java.util.concurrent.TimeUnit
 
-class RandomScan : Madara(
-    "Random Scan",
-    "https://randomscanlators.net",
+class LuraToon : Madara(
+    "LuraToon",
```

Comment:
When doing a name change, you need to preserve the old source's `id` so library entries can be migrated automatically.

Look for the source's `id` [here](https://raw.githubusercontent.com/keiyoushi/extensions/repo/index.json).